audio-worklet/noisegate-test.html

```diff
@@ -42,13 +43,13 @@
         // The threshold should be set slightly above the oscillator's gain, so
         // that the noise gate mutes the signal.
         const oscillatorGain = 1;
-        const threshold = 10 * Math.log10(oscillatorGain * oscillatorGain)
+        const threshold = NoiseGate.toDecibel(oscillatorGain * oscillatorGain)
             + 0.015;
         
         let context = setupThresholdTestGraph(threshold, oscillatorGain);
 
         // The noise gate should never open since the envelope follower won't
-        // quite reach 0 dB.
+        // quite reach the threshold, set slightly above 0 dB.
```

Comment:
Nit:  Move this comment 56.  As it is, it looks like it's documenting startRendering, and not the expected output value.